cosmic-comp-config/src/input.rs

```diff
@@ -29,6 +31,10 @@ pub struct InputConfig {
     pub tap_config: Option<TapConfig>,
     #[serde(skip_serializing_if = "Option::is_none", default)]
     pub map_to_output: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none", default)]
+    pub focus_follows_cursor: Option<bool>,
+    #[serde(skip_serializing_if = "Option::is_none", default)]
+    pub cursor_follows_focus: Option<bool>,
 }
```

Comment:
This is unused. When I initially started I wasn't sure if it should where the setting should go. I left it here in case it needed to be input device or seat specific. I'll remove it in my next push